--- archives/zlib-1.2.8/CMakeLists.txt	2013-04-29 07:57:10.000000000 +0900
+++ sources/zlib-1.2.8/CMakeLists.txt	2015-03-28 23:12:26.000000000 +0900
@@ -5,6 +5,9 @@
 
 set(VERSION "1.2.8")
 
+# to distinguish between debug and release lib
+set(CMAKE_DEBUG_POSTFIX "_d")
+
 option(ASM686 "Enable building i686 assembly implementation")
 option(AMD64 "Enable building amd64 assembly implementation")
 
@@ -213,7 +216,7 @@
 if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL )
     install(TARGETS zlib zlibstatic
         RUNTIME DESTINATION "${INSTALL_BIN_DIR}"
-        ARCHIVE DESTINATION "${INSTALL_LIB_DIR}"
+        ARCHIVE DESTINATION "${INSTALL_LIB_DIR}/static"
         LIBRARY DESTINATION "${INSTALL_LIB_DIR}" )
 endif()
 if(NOT SKIP_INSTALL_HEADERS AND NOT SKIP_INSTALL_ALL )
@@ -247,3 +250,36 @@
     target_link_libraries(minigzip64 zlib)
     set_target_properties(minigzip64 PROPERTIES COMPILE_FLAGS "-D_FILE_OFFSET_BITS=64")
 endif()
+
+
+#============================================================================
+# Contrib libraries
+#============================================================================
+set(MINIZIP_SRCS
+  contrib/minizip/ioapi.c
+  contrib/minizip/miniunz.c
+  contrib/minizip/minizip.c
+  contrib/minizip/unzip.c
+  #contrib/minizip/mztools.c
+)
+
+set(MINIZIP_PUBLIC_HDRS
+    contrib/minizip/crypt.h
+    contrib/minizip/ioapi.h 
+    contrib/minizip/iowin32.h 
+    contrib/minizip/unzip.h
+    contrib/minizip/zip.h
+)
+
+add_library(minizip ${MINIZIP_SRCS})
+
+add_library(minizip ${MINIZIP_SRCS})
+
+install(FILES ${MINIZIP_PUBLIC_HDRS} DESTINATION include)
+
+install(
+    TARGETS minizip
+    LIBRARY DESTINATION lib 
+    ARCHIVE DESTINATION lib/static
+)
+
